software/applications/HelloIOIO/build.gradle

```diff
@@ -3,16 +3,21 @@ apply plugin: 'com.android.application'
 buildscript {
   repositories {
     mavenCentral()
+      google()
+      jcenter()
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.3.3'
+    classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
Btw, it's a bad praxis to specify `com.android.tools.build:gradle:xxx` for each project/module.
It's much better to specify it just once in topmost `build.gradle`

Same to 

```
buildscript {
  repositories {
    mavenCentral()
      google()
      jcenter()
  }
```